src/dymaptic.GeoBlazor.Core/dymaptic.GeoBlazor.Core.csproj

```diff
@@ -69,36 +69,37 @@
 
     <ItemGroup>
         <Folder Include="wwwroot\assets" />
+        <Content Remove="node_modules\**" />
     </ItemGroup>
 
-    <Target Name="NPM Install" BeforeTargets="PreBuildEvent" Condition="$(TargetFrameworks.StartsWith($(TargetFramework)))">
+    <Target Name="NPM Install" BeforeTargets="PreBuildEvent" Condition="$(TargetFrameworks.StartsWith($(TargetFramework))) AND '$(OptOutFromCoreEsBuild)' != 'true'">
```

Comment:
does OptOutFromCoreEsBuild need to be set anywhere in this project? or will it default to true?

Reply:
It defaults to `false`, which we want here because normally you want all this to run. It is only `true` when running from the Pro repository, where this will be set to `true` to skip these steps.